content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -1,9 +1,8 @@
 # Add a model
```

Comment:
The video included in this page (https://www.youtube.com/watch?v=szzv4ZXmYAs) must be updated, because `path to dvc.yaml` is missing in the video and the outdated `artifacts.yaml` is mentioned. @aguschin are there any other changes needed in the video?